api/src/main/resources/client_api.yaml

```diff
@@ -148,11 +150,16 @@ definitions:
         description: >
           A query specifying how to pull data out of a single table.
         $ref: "#/definitions/TableQuery"
+      annotationQuery:
+        description: >
+          A query specifying how to retrieve annotation values created about participants in a
```

Comment:
This cannot be used in combination with `tableQuery`, right? Should document this.

Reply:
Done.